src/format/typst/format-typst.ts

```diff
@@ -40,8 +44,8 @@ export function typstFormat(): Format {
       standalone: true,
       [kDefaultImageExtension]: "svg",
       [kWrap]: "none",
-      [kCiteproc]: false,
     },
+    resolveFormat: typstResolveFormat,
     formatExtras: (
```

Comment:
I choose to do that in `resolveFormat` because `formatExtras` does not allow to modify extensions on the format with `kVariants`. 

I don't think we have another way to conditionally set a Pandoc's extensions based on a metadata information. Is there any other example ?

Reply:
No, I think that's the right way